src/RelSeries.jl

```diff
@@ -29,6 +29,10 @@ base_ring_type(::Type{<:SeriesRing{T}}) where T <: RingElement = parent_type(T)
 
 base_ring(R::SeriesRing{T}) where T <: RingElement = R.base_ring::parent_type(T)
```

Comment:
Is is really correct that we access `R.base_ring` for `R` of type `SeriesRing`?

Reply:
To note down what I just said in triage: no this is not correct, but a left-over from old days; on occasion we find more of these (as you just did) and try to fix them. In this case it might be trivial (just remove the line), but it might also break implementations in e.g. Nemo (though on a quick glance, it *might* be fine there.

Perhaps you can open a separate PR where you just try to delete that line and we can see what happens. However my guess is one will also have to add new `base_ring` methods to `src/generic/AbsMSeries.jl`, `src/generic/AbsSeries.jl`, `src/generic/RelSeries.jl`.